lib/storage.js

```diff
@@ -104,28 +110,27 @@ async function getSource(paramsSource, inDirectory, disableSourceDownload) {
     return getAsset(paramsSource, inDirectory, name, disableSourceDownload);
 }
 
-async function putRendition(rendition) {
+async function putRendition(rendition, directories) {
     // Note: validation has happened in validate.js before
 
     if (process.env.WORKER_TEST_MODE) {
-        if(rendition.name.includes('post-')) {
-            // to verify post-processing renditions, since their names contain `post-` in the beginning
-            // and asset-compute-cli command run-worker only looks at the file named `rendition*.*`
-            rendition.name = rendition.name.substring(5);
-            
-            const newPath = path.join(rendition.directory, rendition.name);
-            await fs.rename( rendition.path, newPath);
-            
-            rendition.path = newPath;
+        if (rendition.directory === directories.postprocessing) {
+            // move the post processing rendition to the rendition path as expected by "aio asset-compute test-worker"
+            const newRendition = new Rendition(rendition.instructions, directories.out, rendition.index);
+
+            console.log(`WORKER_TEST_MODE: moving ${rendition.path} to ${newRendition.path}`);
```

Comment:
idea: maybe worker_test_mode also gets added to the console log prefix?

Reply:
IMO that would be a bit long and make it harder to read the `build/test.log`... I only added it where something special is happening in test mode, and to follow the files around (because this has become non-trivial now with all the post processing changes).

Reply:
Also the setConsoleLogPrefix() does not support nested prefixes, that would be more work to get right.

Reply:
Also, it is easy to know when we are running tests, and when we look at production logs. So there is not a lot of added value to do that here and now either.